docarray/array/storage/qdrant/helper.py

```diff
@@ -22,4 +24,6 @@ def embedding_to_array(
         if embedding.ndim > 1:
             embedding = np.asarray(embedding).squeeze()
 
+        if np.all(embedding == 0):
+            embedding = embedding + EPSILON
```

Comment:
Isnt this a little expensive? Also in some cases embeddings are not even numpy arrays and may live in GPU

Reply:
Qdrant does not accept null vectors because cosine distance can't be computed
Basicly we have 2 options:

- prohibit users from using null vectors as embedding
- implicitly add Epsilon to allow computation
your thoughts ?

Reply:
I prefer the second one, as same as we have done in weaviate.